pom.xml

```diff
@@ -274,7 +280,7 @@
     <profile>
       <id>fitnesse-integration</id>
       <activation>
-        <activeByDefault>true</activeByDefault>
+        <activeByDefault>false</activeByDefault>
```

Comment:
Why this change? 

When I try to upgrade FitNesse, I get an error:

```
     [java] **********************************************************
     [java] Executing command: ProjectXebium?suite&format=text&excludeSuiteFilter=NotOnPhantomJS
     [java] -----Command Output-----
     [java] Exception in thread "main" java.lang.AbstractMethodError
     [java]     at fitnesse.wikitext.parser.ParseSpecification.parse(ParseSpecification.java:96)
     [java]     at fitnesse.wikitext.parser.Parser.parse(Parser.java:141)
     [java]     at fitnesse.wikitext.parser.Parser.parse(Parser.java:137)
```

We should fix that error instead of disabling the tests :)